examples/multi-runner/main.tf

```
@@ -3,7 +3,27 @@ locals {
   aws_region  = "eu-west-1"
 
   # Load runner configurations from Yaml files
-  multi_runner_config = { for c in fileset("${path.module}/templates/runner-configs", "*.yaml") : trimsuffix(c, ".yaml") => yamldecode(file("${path.module}/templates/runner-configs/${c}")) }
+  multi_runner_config_files = {
+    for c in fileset("${path.module}/templates/runner-configs", "*.yaml") :
+
+    trimsuffix(c, ".yaml") => yamldecode(file("${path.module}/templates/runner-configs/${c}"))
+  }
+  multi_runner_config = {
+    for k, v in local.multi_runner_config_files :
+
+    k => merge(
+      v,
+      {
+        runner_config = merge(
+          v.runner_config,
+          {
+            subnet_ids = lookup(v.runner_config, "subnet_ids", null) != null ? module.base_other.vpc.private_subnets : null
+            vpc_id     = lookup(v.runner_config, "vpc_id", null) != null ? module.base_other.vpc.vpc_id : null
```

Example of a second VPC is nice, but this is our default example to test. Would prefer we can keep it if one vpc. I think the suggestion below hows clearly how to use it. And even test it by injecting a single subnet.

```suggestion
            subnet_ids = lookup(v.runner_config, "subnet_ids", null) != null ? [module.base.vpc.private_subnets[0]] : null
            vpc_id     = lookup(v.runner_config, "vpc_id", null) != null ? module.base.vpc.vpc_id : null
```